src/utils.ts

```diff
@@ -0,0 +1,11 @@
+import { ServerResponse } from 'node:http';
+
+import { green, red, yellow } from 'colorette';
+
+export const isWarnStatus = (res: ServerResponse) => res.statusCode >= 400 && res.statusCode < 499;
+export const isErrorStatus = (res: ServerResponse) => res.statusCode >= 500 && res.statusCode < 599;
```

Comment:
```suggestion
export const isWarnStatus = (res: ServerResponse) => res.statusCode >= 400 && res.statusCode <= 499;
export const isErrorStatus = (res: ServerResponse) => res.statusCode >= 500 && res.statusCode <= 599;
```
seems like we want this to be inclusive for both ranges?

Reply:
yes definitely 😓